wiki/FAQ/en.md

```diff
@@ -395,16 +422,16 @@ The falling notes must be tapped on the judgement line, with correct key corresp
 More keys corresponding to the falling notes must be tapped simultaneously if the notes fall simultaneously.
 A score burst will be given when tapped a correct key corresponding to the note in the judgement line.
 
-##### Holdnotes
-![osu!mania's holdnotes][osu!mania holdnotes image]
+##### Hold notes
+![osu!mania's hold notes][osu!mania holdnotes image]
 
-_A typical play with some holdnotes_
+_A typical play with some hold notes_
 
-Holdnotes are the sliders and spinners of osu!mania.
-When the holdnote reaches the judgement line, tap the starting note in time with correct key, hold, and release it at the ending note of the holdnote.
+Hold notes are the sliders and spinners of osu!mania.
+When the hold note reaches the judgement line, tap the starting note in time with correct key, hold, and release it at the ending note of the hold note.
 
-Healthbar regeneration occurs slowly while holding the holdnote(s).
-Depending on the song and mapping quirks, it is possible that other notes or holdnotes to appear while holding the previous holdnote(s).
+Healthbar regeneration occurs slowly while holding the hold note(s).
+Depending on the song and mapping quirks, it is possible that other notes or hold notes to appear while holding the previous hold note(s).
 
 #### Controls
 ![Default Key layouts of osu!mania (Options-&gt;Input)][osu!mania key layout image]
```

Comment:
Even with a high quality image, I _will_ use _the_ **MS Paint** to do it.

Gonna skip this one out since it is one of my weakest expertise.